client/blocks/reader-post-card/style.scss

```diff
@@ -400,6 +406,13 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 }
 
+// 3 line excerpt for thumbnail cards
+.reader-post-card.card.has-thumbnail:not(.is-gallery) {
```

Comment:
would it be better to make this without any extra qualifiers and then delete line#377?
Because 3 is the general case and then 2 is an edge case.

```css
.reader-post-card__excerpt {
   max-height: 15px * 1.5 * 3;
}
```